README.md

```diff
@@ -13,7 +13,7 @@ cd FAIR-MAST-Dashboard
 ```bash
 python -m venv venv
 . venv/bin/activate
-pip install s3fs 'zarr < 3' panel watchfiles holoviews hvplot xarray
+pip install holoviews hvplot panel s3fs watchfiles xarray zarr
```

Comment:
```suggestion
pip install holoviews hvplot panel s3fs watchfiles xarray 'zarr >= 3'
```

Reply:
> Sorry for the delayed response!

No worries!

> I imagine older major versions of some of the other dependencies wouldn't work either...

Indeed, I'd expect so as well. With this reasoning I'm happy to merge as is :)